test/server/server_test.cc

```diff
@@ -476,8 +476,7 @@ class TestWithSimTimeAndRealSymbolTables : public Event::TestUsingSimulatedTime
 class ServerStatsTest : public TestWithSimTimeAndRealSymbolTables, public ServerInstanceImplTest {
 protected:
   void flushStats() {
-    // Default flush interval is 5 seconds.
-    simTime().sleep(std::chrono::seconds(6));
+    server_->flushStats();
```

Comment:
Can you make sure there are tests that cover both flushing stats by advancement of time, and also flushing stats via explicit call?

Reply:
oops sorry, commented before pushing. Yep I am just rejiggering the inheritance so I can parametrized that test. Great call out!